.github/workflows/clang-tidy.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
         run: |
           apt-get install -y \
             clang \
+            libstdc++-15-dev \
```

Comment:
Why do we need this? Doesn't it come with clang? Do we need to update docs - readme and install.rst?

Reply:
clang supports two incompatible C++ libraries: `libc++` and `libstdc++`. `libc++` is part of the clang ecosystem and used by default. Debian packages, including Qt, are compiled against the latest `libstdc++`. CMake correctly selects `libstdc++`, but it's no longer installed by default when pulling clang.